kubernetes-dashboard-api.yaml

```diff
@@ -0,0 +1,43 @@
+package:
+  name: kubernetes-dashboard-api
+  version: 1.10.1
+  epoch: 0
+  description: Go module handling authentication to the Kubernetes API
+  copyright:
+    - license: Apache-2.0
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/kubernetes/dashboard
+      tag: api/v${{package.version}}
+      expected-commit: 8c15a76aec0489f63ab841e4aaf09391d2e68912
+
+  - uses: go/build
+    with:
+      packages: .
+      modroot: ./modules/api
+      output: dashboard-api
+
+  - uses: strip
+
+test:
+  environment:
+    contents:
+      packages:
+        - curl
+  pipeline:
+    - uses: test/kwok/cluster
+    - name: Verify kubernetes-dashboard-api installation
+      runs: |
+        dashboard-api  --bind-address=0.0.0.0 --kubeconfig=/root/.kube/config &> dashboard-api.log &
+        sleep 5
+        grep "Starting Kubernetes Dashboard API" dashboard-api.log || exit 1
+
+update:
+  enabled: true
+  github:
+    identifier: kubernetes/dashboard
+    strip-prefix: api/v
+    use-tag: true
```

Comment:
upstream is doing active releases as well so we can consider using releases as well instead of tags and that'll filter out any beta|rc releases. but this looks good as well given the project is matured. not a blocker. Feel free to skip.

Reply:
I think using tags should be good enough.